App/arca/assets/css/recoveryPass.scss

```diff
@@ -0,0 +1,141 @@
+$main-color: #483D8B;
+
+body{
+    background: #FBFCFC;
+}
+.con-img-pass img{
+    height: 100vh;
+}
+.con-recovery-password{
+    display: grid;
+    grid-template-columns: 40% 60%;
+    grid-template-rows: 100%;
+    justify-content: center;
+    row-gap: 30px;
+    height: calc(100vh - 50px);
+}
+.head-recovery{
+    display: grid;
+    grid-template-columns: 80%;
+    grid-template-rows: 20% 10% 10% 10%;
+    row-gap: 50px;
+    padding: 40px;
+    padding-right: 180px;
+    align-content: center;
+    justify-content: end;
+    justify-self: end;
+}
+.con-form-recoverypass{
+    display: grid;
+    grid-template-columns: 70%;
+    justify-content: end;
+
+}
+.head-1 h2{
+    font-family: 'Poppins', sans-serif;
+    font-size: 65px;
+    font-weight: bold;
+    text-align: center;
+    margin-top: 10px;
+    text-align: end;
+}
```

Comment:
This is pretty similar to the previous file, I suggest creating a file for the typography definitions, I mean, how should looks all titles, all paragraphs, labels or other texts